src/react-components/inputs/input-with-suggestions.jsx

```diff
@@ -14,46 +14,57 @@
  */
 
 import React, { useState, useEffect } from 'react';
-import Input from './input';
+import Input from '../../react-components/inputs/input';
 
 
-const InputWithSuggestions = ({ data, onClick, sortFunction, displayValue, ...props }) => {
+const InputWithSuggestions = ({ data, onClick, displayValue, placeholder, ...props }) => {
+
+    const filterValues = (values, filterValue) => {
```

Comment:
Can we shorten this line with return statement and rewrite this part of code as: 
 ```
    const filterValues = (values, filterValue) => {
        return filterValue !== '' ? values.filter((element) => { 
            return element.name.toLowerCase().indexOf(filterValue.toLowerCase()) > -1; 
        }) : values;
    };
```